java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -1766,34 +1806,74 @@ public CompletableFuture<Map<String, Double>> zpopmin(@NonNull String key, long
                 ZPopMin, new String[] {key, Long.toString(count)}, this::handleMapResponse);
     }
 
+    @Override
+    public CompletableFuture<Map<GlideString, Double>> zpopmin(@NonNull GlideString key, long count) {
```

Comment:
shouldn't we change it to zpopmincount?

Reply:
Why should we add the suffix? This should be same as the String version. Same below

Reply:
What I am saying is that we might have this error in the string format and you copied it.
I will approve but this needs to be discussed